### PR TITLE
Add update document endpoint

### PR DESCRIPTION
**Pull request by TayGov**
# Description

This PR includes the following proposed change(s):

- {List all the changes, if possible add the jira ticket #}

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Does the change impact or break the Docker build?

- [ ] Yes
- [x] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [ ] No

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/backend/efiling-reviewer-api/src/test/java/ca/bc/gov/open/jag/efilingreviewerapi/documentconfiguration/GetDocumentConfigurationsTest.java [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+import org.springframework.http.ResponseEntity;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+public class GetDocumentConfigurationsTest {
+    private static final String CASE_1 = "CASE1";
+    private static final String CASE_2 = "CASE2";
+    public static final String DESCRIPTION = "DESCRIPTION";
```
Comment (karthikmsqa) [inline src/backend/efiling-reviewer-api/src/test/java/ca/bc/gov/open/jag/efilingreviewerapi/documentconfiguration/GetDocumentConfigurationsTest.java:23]: nit: Should this be a private field?

File: src/backend/efiling-reviewer-api/src/test/java/ca/bc/gov/open/jag/efilingreviewerapi/documentconfiguration/CreateDocumentConfigurationsTest.java [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+import org.mockito.MockitoAnnotations;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+public class CreateDocumentConfigurationsTest {
+    private static final String CASE_1 = "CASE1";
+    private static final String CASE_2 = "CASE2";
+    public static final String DESCRIPTION = "DESCRIPTION";
```
Comment (karthikmsqa) [inline src/backend/efiling-reviewer-api/src/test/java/ca/bc/gov/open/jag/efilingreviewerapi/documentconfiguration/CreateDocumentConfigurationsTest.java:22]: nit: Should this be a private field?

**Review by karthikmsqa — COMMENTED** [review]
Couple small clarifications.

File: src/backend/efiling-reviewer-api/src/test/java/ca/bc/gov/open/jag/efilingreviewerapi/documentconfiguration/GetDocumentConfigurationsTest.java (hunk 1 above)
Comment (TayGov) [reply src/backend/efiling-reviewer-api/src/test/java/ca/bc/gov/open/jag/efilingreviewerapi/documentconfiguration/GetDocumentConfigurationsTest.java:23]: Yes it should

**Review by karthikmsqa — APPROVED** [review]
Awesome, thanks @TayGov!

**Pull request merged**